docs/rules/no-useless-computed-key.md

```diff
@@ -1,4 +1,4 @@
-# Disallow unnecessary computed property keys on objects (no-useless-computed-key)
+# Disallow unnecessary computed property keys (no-useless-computed-key)
```

Comment:
I think that it would be better to add 'and classes' instead of removing 'objects', because there is also computed property access, covered by a different rule (`dot-notation`).

It would be good to also make a similar change in the rule's meta `docs.description` field. That's the text used for the list of [all rules](https://eslint.org/docs/rules/).